.github/workflows/ci-pullrequest-main.yml

```diff
@@ -74,4 +76,11 @@ jobs:
       uses: actions/upload-artifact@v3.0.0
       with:
         name: mutation-report
-        path: src/StrykerOutput
\ No newline at end of file
+        path: src/StrykerOutput
+
+    - uses: actions/checkout@v3
```

Comment:
Is this a duplicate? the same action is called on line 14

Reply:
Hmmm - I stole that from the main branch publishing workflow.  Does look weird now you mention it.